From 0859a87d33bfe54143bb60ee75b663d179980224 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Fri, 21 Oct 2022 15:48:09 -0600
Subject: [PATCH] Update setup.py

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 1e75e78..5db2f3d 100644
--- a/setup.py
+++ b/setup.py
@@ -8,9 +8,9 @@
 setup(
     name="emmet-builders",
     use_scm_version={
-        "root": "..",
+        "root": ".",
         "relative_to": __file__,
-        "write_to": "emmet-builders/_version.py",
+        "write_to": "emmet/builders/_version.py",
         "write_to_template": '__version__ = "{version}"',
         "fallback_version": fallback_version,
     },
